database/create_tables.sql

```diff
@@ -39,35 +40,31 @@ CREATE TABLE Transactions (
 );
 
 CREATE TABLE Creates (
-    project_id int REFERENCES Projects(project_id),
+    project_name int REFERENCES Projects(project_name),
     user_id int REFERENCES Users(user_id),
+    create_date datetime NOT NULL,
     CONSTRAINT creates_constraint PRIMARY KEY(user_id, project_id)
 );
 
 CREATE TABLE SearchHistory (
-    search_id serial PRIMARY KEY,
+    search_timestamp timestamp,
```

Comment:
Actually there's an issue here? 
If 2 users search something at the same time then the timestamp will be the same and thus cannot store both searches with different text and same timestamp?

Reply:
Oh, wait. I forgot to add in the `user_id` in the primary key. Thanks for checking!